CHANGELOG.next.toml

```
@@ -10,3 +10,8 @@
 # references = ["smithy-rs#920"]
 # meta = { "breaking" = false, "tada" = false, "bug" = false }
 # author = "rcoh"
+[[smithy-rs]]
+message = "Made fluent operation structs cloneable"
+references = ["aws-sdk-rust#254"]
+meta = { "breaking" = false, "tada" = false, "bug" = false }
+author = "Jacco"
```

this will render it into both changelogs since this is relevant for SDK consumers as well
```suggestion
[[smithy-rs]]
message = "Made fluent operation structs cloneable"
references = ["aws-sdk-rust#254"]
meta = { "breaking" = false, "tada" = false, "bug" = false }
author = "Jacco"

[[aws-sdk-rust]]
message = "Made fluent operation structs cloneable"
references = ["aws-sdk-rust#254"]
meta = { "breaking" = false, "tada" = false, "bug" = false }
author = "Jacco"
```